src/jsonrpc/miner_jsonrpc_dkg.erl

```diff
@@ -13,13 +13,13 @@
 %%
 
 handle_rpc(<<"dkg_status">>, []) ->
-    Status = try
-                 miner_consensus_mgr:dkg_status() of
-                 Stat -> Stat
-             catch _:_ ->
-                       <<"not_running">>
-             end,
-    #{ status => Status };
+    try
+        miner_consensus_mgr:dkg_status() of
+            not_running -> #{ running => false };
+            Status -> #{ running => true, status => Status }
+    catch _:_ ->
+        #{ running => false }
+    end;
```

Comment:
This is a return-type normalization. There will always be a `running` attribute to check.